deploy/olm-catalog/akka-cluster-operator/0.2.1/akka-cluster-operator-certified.v0.2.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,332 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: akka-cluster-operator-certified.v0.2.1
+  namespace: placeholder
+  annotations:
+    capabilities: Seamless Upgrades
+    categories: Application Runtime
+    certified: 'false'
```

Comment:
Should this be true?

Reply:
The folks at Red Hat tell me it doesn't matter, so I'm not sure why it's there. 
Updated to true cuz it is. Thanks.